ui/admin/tests/acceptance/aliases/selectors.js

```diff
@@ -0,0 +1,7 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+export const DESTINATION_ID_SELECTOR = '[name="destination_id"]';
```

Comment:
If these selectors refers to a field, the pattern to follow will be FIELD_[NAME_OF_THE_FILED] , as examples you can check [accounts selector file](https://github.com/hashicorp/boundary-ui/blob/main/ui/admin/tests/acceptance/accounts/selectors.js#L18-L28) or [auth-method seelctors file](https://github.com/hashicorp/boundary-ui/blob/main/ui/admin/tests/acceptance/auth-methods/selectors.js#L6-L97)